### ISSUE TITLE
Various guidelines: eliminate violations (2-4 issues per guideline)

### ISSUE THREAD

**Issue by PhilippSalvisberg** [Trivadis/plsql-and-sql-coding-guidelines#194]
There are between 2 and 4 violations for various guidelines in [SonarCloud](https://sonarcloud.io/project/issues?resolved=false&rules=external_db*+CODECOP%3AG-1010%2Cexternal_db*+CODECOP%3AG-1030%2Cexternal_db*+CODECOP%3AG-1070%2Cexternal_db*+CODECOP%3AG-2180%2Cexternal_db*+CODECOP%3AG-2185%2Cexternal_db*+CODECOP%3AG-2220%2Cexternal_db*+CODECOP%3AG-2230%2Cexternal_db*+CODECOP%3AG-2510%2Cexternal_db*+CODECOP%3AG-3140%2Cexternal_db*+CODECOP%3AG-3145%2Cexternal_db*+CODECOP%3AG-3150%2Cexternal_db*+CODECOP%3AG-3180%2Cexternal_db*+CODECOP%3AG-3190%2Cexternal_db*+CODECOP%3AG-4220%2Cexternal_db*+CODECOP%3AG-4330%2Cexternal_db*+CODECOP%3AG-4370%2Cexternal_db*+CODECOP%3AG-4375%2Cexternal_db*+CODECOP%3AG-4380%2Cexternal_db*+CODECOP%3AG-4385%2Cexternal_db*+CODECOP%3AG-4390%2Cexternal_db*+CODECOP%3AG-4395%2Cexternal_db*+CODECOP%3AG-5040%2Cexternal_db*+CODECOP%3AG-5060%2Cexternal_db*+CODECOP%3AG-5070%2Cexternal_db*+CODECOP%3AG-7120%2Cexternal_db*+CODECOP%3AG-7130%2Cexternal_db*+CODECOP%3AG-7150%2Cexternal_db*+CODECOP%3AG-7160%2Cexternal_db*+CODECOP%3AG-7220%2Cexternal_db*+CODECOP%3AG-7230%2Cexternal_db*+CODECOP%3AG-7310%2Cexternal_db*+CODECOP%3AG-7410%2Cexternal_db*+CODECOP%3AG-7430%2Cexternal_db*+CODECOP%3AG-7710%2Cexternal_db*+CODECOP%3AG-7730%2Cexternal_db*+CODECOP%3AG-7910%2Cexternal_db*+CODECOP%3AG-8110%2Cexternal_db*+CODECOP%3AG-9040%2Cexternal_db*+CODECOP%3AG-9113%2Cexternal_db*+CODECOP%3AG-9114&id=Trivadis-plsql-and-sql-coding-guidelines).

Basically just the bad example of a guideline should be recorded as violation. All other SQL code blocks should honor this rule and should be changed.

**Comment by PhilippSalvisberg** [comment]
closed via PR #199
